rmw_connext_cpp/src/functions.cpp

```diff
@@ -1124,8 +1134,10 @@ rmw_destroy_subscription(rmw_node_t * node, rmw_subscription_t * subscription)
   return RMW_RET_OK;
 }
 
+RMW_LOCAL
```

Comment:
I just go rid of it, since the default visibility should be hidden anyways (this is done by adding a compiler flag in CMake). 0488637